src/getHumanAge.test.js

```diff
@@ -7,4 +7,44 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array with two elements', () => {
+    expect(getHumanAge(10, 4))
+      .toHaveLength(2);
+  });
+
+  test('should return 0, 0 if cat years < 15 and dog years < 15', () => {
+    expect(getHumanAge(0, 14))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 1, 1 if cat and dog years >= 15 and <= 23', () => {
```

Comment:
Test description is not accurate. Better to say 'in range' for instance. Try improve description in all tests